lte/gateway/c/core/BUILD.bazel

```diff
@@ -457,13 +457,13 @@ cc_library(
         "oai/tasks/s11/s11_mme_bearer_manager.c",
         "oai/tasks/s11/s11_mme_session_manager.c",
         "oai/tasks/s11/s11_tasks.c",
-        "oai/tasks/s1ap/s1ap_mme.c",
-        "oai/tasks/s1ap/s1ap_mme_decoder.c",
-        "oai/tasks/s1ap/s1ap_mme_encoder.c",
-        "oai/tasks/s1ap/s1ap_mme_handlers.c",
-        "oai/tasks/s1ap/s1ap_mme_itti_messaging.c",
-        "oai/tasks/s1ap/s1ap_mme_nas_procedures.c",
-        "oai/tasks/s1ap/s1ap_mme_ta.c",
+        "oai/tasks/s1ap/s1ap_mme.cpp",
+        "oai/tasks/s1ap/s1ap_mme_decoder.cpp",
+        "oai/tasks/s1ap/s1ap_mme_encoder.cpp",
+        "oai/tasks/s1ap/s1ap_mme_handlers.cpp",
+        "oai/tasks/s1ap/s1ap_mme_itti_messaging.cpp",
+        "oai/tasks/s1ap/s1ap_mme_nas_procedures.cpp",
+        "oai/tasks/s1ap/s1ap_mme_ta.cpp",
```

Comment:
Can we also rename the `s1ap_types.h` entry in line 596 to `s1ap_types.hpp` in this `BUILD.bazel` file to pass the [C/C++ bazelification check](https://github.com/magma/magma/runs/6504759864?check_suite_focus=true)?